terraform/gcp/locals.tf

```diff
@@ -0,0 +1,5 @@
+locals {
+  buz_sha               = "sha256:1083c0333c284dfa16dd7cc621f90b8a1197fe4d9905237e41f2f1a495481d92"
```

Comment:
Don't love this. Reading up on GCP there isn't a data source in Terraform for the images within artifact registry (though there is for container registry which is being deprecated). Don't know the best way to reference the image in the Cloud Run service.

Reply:
👀